build.gradle.kts

```diff
@@ -36,3 +36,19 @@ val detektAll = tasks.register<Detekt>("detektAll") {
         html.enabled = false
     }
 }
+
+tasks.register<Exec>("installGitHooks") {
+    description = "Install local git hooks"
+    group = "Build Setup"
+
+    commandLine("chmod")
+    args("-R", "u+x", ".githooks")
```

Comment:
git remembers file attributes anyway, why do we need to explicitly set them manually?